### ISSUE TITLE
230822/ pickle 

### ISSUE THREAD

**Issue by doxgxxn** [doxgxxn/WILT#15]
Sample code for Iris:
    
    from sklearn.datasets import load_iris
    from sklearn.model_selection import train_test_split
    from sklearn.metrics import accuracy_score
    from autosklearn.classification import AutoSklearnClassifier
    import pickle
    
    
    # dataset:
    X, y = load_iris(return_X_y=True)
    X_train, X_test, y_train, y_test = train_test_split(X, y, test_size = 0.2)
    
    # train model:
    classifier = AutoSklearnClassifier(
        time_left_for_this_task=30, 
        per_run_time_limit=60,
        memory_limit=1024*12) # depends on your computer
    classifier.fit(X_train, y_train)
    
    # save model
    with open('iris-classifier.pkl', 'wb') as f:
        pickle.dump(classifier, f)
    
    # load model
    with open('iris-classifier.pkl', 'rb') as f:
        loaded_classifier = pickle.load(f)
    
    # predict
    y_true = y_test
    y_pred = loaded_classifier.predict(X_test)
    print('iris classifier: accuracy:', accuracy_score(y_true, y_pred))
    # iris classifier: accuracy: 0.9333333333333333
